theoriq/api/v1alpha2/schemas/schemas.py

```diff
@@ -17,6 +17,10 @@ class ExecuteSchema(BaseModel):
     request: Dict[str, Any]
     response: Dict[str, Any]
 
+    @classmethod
+    def from_base_models(cls, *, request: Type[BaseModel], response: Type[BaseModel]) -> ExecuteSchema:
+        return cls(request=request.model_json_schema(), response=response.model_json_schema())
```

Comment:
This should be something closer to:
```suggestion
        return cls(request=RootModel[List[Union[request.model_json_schema()]]], response=RootModel[List[Union[response.model_json_schema()])
```

where `RootModel` comes from Pydantic.
Here is an example of schema and validated json content: https://www.jsonschemavalidator.net/s/lp4Lkh14